README.md

```diff
@@ -158,15 +158,16 @@ prettier.format(YOUR_CODE, {
 
 ## Configuration
 
-Prettier for PHP supports the following options:
+Prettier for PHP supports the following options. We recommend that all users set the `phpVersion` option.
```

Comment:
I think it is unnecessary, prettier is 0CJS, i think we should try to load `composer.json` and resolve version automatically, also default value  should be `7.2` because it is latest LTS

Reply:
Hmm, we've had quite a long discussion about this and decided differently. Please see #1280 and #1274.

Reply:
Also, looking up the project's `composer.json` is insufficient. It may easily be possible that a project has dependencies installed which require a higher PHP version. If we'd want to do it, we'd have to run `composer why php`, parse the output columns and analyse version constraints to pick the lowest (or rather, highest) common denominator.

Reply:
:+1: 
Also, running the prettier plugin with a lower version setting than what is actually used is not really an issue. One is just missing out on some minor optimizations, but it still "just works".

Reply:
I read them, and a little disagree, in real world we have:
- old projects like `wordpress` (plugins/themes/etc) they still support `5.4`
- new projects uses minimum supported LTS

So using `7.0` is not good idea, it will not suit anyone and will force both groups to setup the option, I think we should free from setuping at least one of the groups

> i think we should try to load composer.json and resolve version automatically

That we can do it in future

Reply:
We went for the `7.0` default because our formatting actually doesn't change all the way down to `5.4`:

> 7.1, 7.0, 5.6, 5.5, 5.4: Since our formatting doesn't change between these versions, all of them are equivalent. Since most users are using one of these versions, this category would be the default.

Reply:
I am afraid for trailing comma, it can break code

Reply:
Trailing commas in arrays and lists are supported since very old PHP version (5.0?) and more trailing commas are only printed when the `phpVersion` option is set to 7.2 or higher, so we should be safe, right?

Reply:
Yes, but in future versions this could potentially be dangerous (like HEREDOC/NOWDOC), if no changed why do not use `5.4` as default value, to avoid misleading for new developers?

Reply:
Yeah, i guess we could also use `5.4` as a default, in a sense of a "lowest common denominator". @loilo what do you think?

Reply:
The only thing I don't like about the `5.4` default is that this version reached its end of life in september 2015 - which might be confusing to people when they expect our tool to be "modern" :wink:

Reply:
We can explain why 5.4 is default :smile:

Reply:
I dislike it, but I don't doubt that it might be reasonable. :)

That would make a feature which parses the Composer dependency tree even more useful though...

Reply:
...that may have not been clear enough: I support the decision for 5.4, I just hate that we have to do it. 😁

Reply:
Sometimes you do what you hate :smile:

Reply:
Alright, fixed :slightly_smiling_face: Turns out we had an mismatch between documented and actual default version before :see_no_evil: